lossy_filters.py

```diff
@@ -0,0 +1,572 @@
+DScale5:
+  filters:
+  - - 6
+    - 1
+    - - 0
+      - 5
+      - 800648
+      - 1
+      - 8
+      - 0
+      - 0
+      - 1
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 1818321779
+      - 1717989221
+      - 7628147
+      - 0
+    - !!binary |
+      c2NhbGVvZmZzZXQ=
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwARID8ACAAAAAAAQAA0CwA0/wMAAA==
+DScale6:
+  filters:
+  - - 6
+    - 1
+    - - 0
+      - 6
+      - 800648
+      - 1
+      - 4
+      - 0
+      - 0
+      - 1
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 1818321779
+      - 1717989221
+      - 7628147
+      - 0
+    - !!binary |
+      c2NhbGVvZmZzZXQ=
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwARIB8ABAAAAAAAIAAXCAAXfwAAAA==
+FMantissa13:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 4
+      - 0
+      - 22
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwARIBUABAAAAAAAFgANCAANfwAAAA==
+FMantissa9:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 4
+      - 0
+      - 18
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwARIBEABAAAAAAAEgAJCAAJfwAAAA==
+HalfFloat:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 4
+      - 0
+      - 16
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwARIA8ABAAAAAAAEAAKBQAKDwAAAA==
+Nbit36:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 8
+      - 0
+      - 36
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwAQCAAACAAAAAAAJAA=
+Nbit40:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 8
+      - 0
+      - 40
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwAQCAAACAAAAAAAKAA=
+Nbit48:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 8
+      - 0
+      - 48
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwAQCAAACAAAAAAAMAA=
+Nbit56:
+  filters:
+  - - 5
+    - 1
+    - - 8
+      - 0
+      - 800648
+      - 1
+      - 8
+      - 0
+      - 56
+      - 0
+    - !!binary |
+      bmJpdA==
+  - - 3
+    - 0
+    - []
+    - !!binary |
+      ZmxldGNoZXIzMg==
+  type: !!binary |
+    AwAQCAAACAAAAAAAOAA=
+"""
+filter_dict = yaml.safe_load(filter_data)
```

Comment:
Could we load this from compression/filters.yml instead of redefining it?

Reply:
Yes, that probably makes sense. SOAP will only run from the source directory at the moment so we can just use a relative path to find it.